less/home-page.less

```diff
@@ -72,4 +145,42 @@
       }
     }
   }
-}// home
\ No newline at end of file
+}// home
+
+// Contact Us styling
+
```

Comment:
Something is going on with this so that I cannot see the form. Make sure you start from the parent-component and nest your way down when you are using LESS.